### DISCUSSION TITLE
SAML and LDAP support (Ory as Identity Provider)?

### DISCUSSION THREAD

**Discussion by trpinnovator** [ory/.github#22] [Q&A]
I would like to use Ory Kratos as my Identity Provider (via Hydra) for several legacy clients that only support SAML/LDAP, e.g. stackstorm, gitea.

- is SAML supported by Hydra? This [issue](https://github.com/ory/hydra/issues/29) says SAML is already supported but I can't find any information on the ory docs.
- is LDAP supported by Hydra? If no, is there an option to still provide SSO for LDAP only clients? Any example implementation would be valuable.

Thanks.

**Comment by vinckr** [comment]
hello @trpinnovator 

SAML:
the linked issue says indeed its now _possible_ by implementing a consent endpoint that understands SAML, however there was [not a lot of interest](https://github.com/ory/hydra/issues/1519) from the community.

You can however use it directly in Ory Kratos (soon), see this PR https://github.com/ory/kratos/pull/2653

LDAP: 
This is still an open issue heavily discussed in Ory Kratos, see here: https://github.com/ory/kratos/issues/274. For Hydra there is a community project, but I dont know how well it works: https://github.com/i-core/werther
